code/modules/planet/planet.dm

```diff
@@ -50,3 +53,17 @@
 	sun["color"] = new_color
 	needs_work |= PLANET_PROCESS_SUN
 
+/datum/planet/proc/build_suns()
+	var/desired_suns_per_row = 15
```

Comment:
Might want to give a comment on how to calculate this number, for maps that aren't the same size as ours?

Reply:
Sadly I don't have a way to determine an ideal number. I got to 15 with trial and error and I'm not sure if its worth it to go even higher or not, however the code will adjust for different map sizes already.

Reply:
Roger.